mk/common.mk

```diff
@@ -37,5 +37,6 @@ $(OUT)/local.mk:
 
 clean:
 	$(RM) $(deps) $(TESTS) $(LIBS) $(OBJS)
+	$(RM) -r $(LIBRABBITMQ_PATH)/build
```

Comment:
Don't mix RabbitMQ specific rules in `common.mk`. Instead, you can call predefined macros or execute target in RabbitMQ-oriented `.mk` file(s).